packages/shared/src/components/modals/common/Prompt.tsx

```diff
@@ -0,0 +1,9 @@
+import React, { ReactElement } from 'react';
+
+export type PromptProps = {
+  title: string;
+};
+
+export function Prompt({ title }: PromptProps): ReactElement {
```

Comment:
Seems like an unnecessary component, is it not?

Reply:
I will update this component, (it is not done yet)